README.md

```diff
@@ -37,10 +37,10 @@ gulp.task('translate', ['build'], function(){
 module.exports = gulp;
 ```
 
-### What happens on disk
+### What happens:
```

Comment:
Unless content relevant, titles to do get line-ending punctuation.

But this title is now a bit usless.  maybe it should be: `Creating language builds`